### PR TITLE
Specify UTC for "last data update" time

### PR DESCRIPTION
**Pull request by jhsoby**

### CODE REVIEW

File: inc/lib/LexemeParty.class.php [hunk 1]
```diff
@@ -614,4 +614,4 @@ public static function displayRankings($rankings, $numberOfConcepts, $anchorLink
     
 }
 
-?>
```
Comment (envlh) [inline inc/lib/LexemeParty.class.php:617]: Can you please remove this change on the last line of the file?

Comment (jhsoby) [reply inc/lib/LexemeParty.class.php:617]: Huh, I don't know why it did that. I just used the built-in editor in GitHub, and didn't touch that line at all. 🧐

Comment (jhsoby) [reply inc/lib/LexemeParty.class.php:617]: It seems the editor automatically adds a newline at the end when you save, and there's no way (at least not that I found) to edit that in the web interface. But isn't newlines at the end the standard any way?

Comment (envlh) [reply inc/lib/LexemeParty.class.php:617]: Even if it's the standard, it should be done in another merge request. I took your commit locally, edited it, and pushed it.

**Comment by envlh** [comment]
Thank you! I merged it manually to avoid the side effect change on the last line.

**Pull request closed without merging**